plutus-core/plc/Main.hs

```diff
@@ -91,6 +93,7 @@ data Input       = FileInput FilePath | StdInput
 data Output      = FileOutput FilePath | StdOutput
 data Language    = TypedPLC | UntypedPLC
 data TimingMode  = NoTiming | Timing Integer deriving (Eq)  -- Report program execution time?
+data CekModel    = Default | Unit   -- Which cost model should we use for CEK machine steps?
```

Comment:
Could we just teach tallying mode to keep a count of how many times it saw each thing? that seems useful in general!

Reply:
Maybe later.  I think that'd take a little time and what's there at the moment does the job, albeit crudely.  It's only used by the `plc` command at the moment, so it's not too important.